src/schema/v2/previewSavedSearch/previewSavedSearch.ts

```diff
@@ -113,7 +113,7 @@ const PreviewSavedSearchType = new GraphQLObjectType<any, ResolverContext>({
   }),
 })
 
-const PreviewSavedSearchAttributesType = new GraphQLInputObjectType({
+export const PreviewSavedSearchAttributesType = new GraphQLInputObjectType({
```

Comment:
**thought (non-blocking):** Wonder if this should have a more generic (non-"preview") name now. But I suppose that would be a breaking change…

Reply:
Agree with you, but maybe I can extract values to a constant with a generic name and reference these values here. I'll do it in another PR